akka-stream/src/main/scala/akka/stream/ActorMaterializer.scala

```diff
@@ -129,12 +129,11 @@ object ActorMaterializer {
    * INTERNAL API
    */
   private[akka] def downcast(materializer: Materializer): ActorMaterializer =
-    materializer match {
+    materializer match { //FIXME this method is going to cause trouble for other Materializer implementations
```

Comment:
in what sense? We use it on places where we need actors or an actor system, these ops won't work without one (can't remember all of them but `log()` *without* providing a `LoggingAdapter` is one usage, with providing one explicitly its ok)